src/docs/portability.dox

```diff
@@ -0,0 +1,26 @@
+/*! @page portability General information on portability
+
+@section cpu Supported CPU architectures
+
+The supported list of CPU target architectures may be found by inspecting the \c WT_ARCH variable
```

Comment:
I considered that. I think the "real" fix there is to remove that code - if it's not in `WT_ARCH`, we can't build for it (since there won't be a `config.cmake` in `cmake/configs/$ARCH/$OS/` that defines sets up the build correctly).

Reply:
We should create a cleanup ticket for this.